metasyn/util.py

```diff
@@ -0,0 +1,136 @@
+"""Configuration classes for creating metaframes."""
```

Comment:
```suggestion
"""Utility module for MetaSyn.

This module provides utility classes that are used across MetaSyn,
including classes for specifying distributions and storing variable
configurations.
"""
```

Not sure if this is a better description, but it's nice to not have it be the exact same as the config.py one